tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -320,6 +308,72 @@ async def test_restarting_f_replicas_for_view_change(self, bft_network):
         await bft_network.wait_for_fast_path_to_be_prevalent(
             run_ops=lambda: skvbc.run_concurrent_ops(num_ops=20, write_weight=1), threshold=20)
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: c == 0, rotate_keys=True)
+    @with_constant_load
```

Comment:
As discussed on our 1:1, `@with_constant_load` and `@verify_linearizability` don't play along nicely. The reason is, the `@with_constant_load` decorator doesn't use the tracker to record write requests, and therefore the linearizability check would fail.

Reply:
PS: if you need `@with_constant_load` for this test scenarion, let's remove the `@verify_linearizability` decorator for now.